frontend/packages/ceph-storage-plugin/src/plugin.ts

```diff
@@ -269,24 +266,6 @@ const plugin: Plugin<ConsumedExtensions> = [
       required: [CEPH_FLAG],
     },
   },
-  {
-    type: 'HorizontalNavTab',
-    properties: {
-      model: NodeModel,
-      page: {
-        href: 'disks',
-        // t('ceph-storage-plugin~Disks')
-        name: '%ceph-storage-plugin~Disks%',
-      },
```

Comment:
Removed whole node disk list page?

Reply:
yes, its another page which overrides lso disk page.
this page has ocs disk rep feature, so removed that, not required.

Reply:
ack